pydoctor/sphinx_ext/build_apidocs.py

```diff
@@ -50,40 +88,70 @@ def on_build_finished(app: Sphinx, exception: Exception) -> None:
         runs = {'main': runs}
 
     for key, value in runs.items():
-        _run_pydoctor(key, value, placeholders)
+        arguments = _get_arguments(value, placeholders)
+
+        options, _ = parse_args(arguments)
+        output_path = pathlib.Path(options.htmloutput)
+        temp_path = pathlib.Path(options.htmloutput + '_pydoctor_temp')
 
+        # Update intersphinx_mapping.
+        pydoctor_url_fragment = config.pydoctor_url_fragment
+        if pydoctor_url_fragment:
+            intersphinx_mapping = config.intersphinx_mapping
+            url = pydoctor_url_fragment.format(**{'rtd_version': rtd_version})
+            intersphinx_mapping[key + '-api-docs'] = (url, str(temp_path / 'objects.inv'))
 
-def _run_pydoctor(name: str, arguments: Iterable[str], placeholders: Mapping[str, str]) -> None:
+        # Build the API docs in temporary path.
+        shutil.rmtree(temp_path, ignore_errors=True)
+        _run_pydoctor(key,  arguments)
+        output_path.rename(temp_path)
+
+def _run_pydoctor(name: str, arguments: Sequence[str]) -> None:
     """
     Call pydoctor with arguments.
 
     @param name: A human-readable description of this pydoctor build.
-    @param arguments: Iterable of arguments used to call pydoctor.
-    @param placeholders: Values that will be interpolated with the arguments using L{str.format()}.
+    @param arguments: Sequence of arguments used to call pydoctor.
```

Comment:
```suggestion
    @param arguments: Command line arguments used to call pydoctor.
```
Now that pydoctor presents the type annotation, we no longer need to include the type in the text itself ;)